addons/mail/static/src/discuss/call/common/thread_model_patch.js

```diff
@@ -18,6 +18,13 @@ const ThreadPatch = {
                 this.store.allActiveRtcSessions.delete(r);
             },
         });
+        /**
+         * @type {0|1|2}
+         * 0 = disabled
+         * 1 = active
+         * 2 = discarded
+         */
```

Comment:
TBH I'd rather to have a "real" enum rather than arbitrary values.

```js
const INDICATOR = Object.freeze({
    DISABLED: "DISABLED",
    ENABLED: "ENABLED",
    DISCARDED: "DISCARDED",
});
```

This makes conditions and assignments much easier to understand and debug.

```js
if (this.fullscreenIndicator === INDICATOR.DISABLED) {}
```